Condition/Condition-Hepatosplenomegaly-Example.json

```diff
@@ -0,0 +1,38 @@
+{
+	"resourceType": "Condition",
+	"id": "Condition-Hepatosplenomegaly-Example",
+	"clinicalStatus": { 
+		"coding": [
+			{ 
+				"system": "http://terminology.hl7.org/CodeSystem/condition-clinical", 
+				"code": "active", 
+				"display": "Active"
+			}
+		]
+	},
+	"verificationStatus": { 
```

Comment:
Can be removed, no details in the test order specify this is a differential diagnosis

Reply:
Changed it to provisional to adhere conformance to the Condition profile. 
Cardinality of 1..1 on Condition.verificationStatus